src/qililab/typings/enums.py

```diff
@@ -211,6 +214,7 @@ class InstrumentControllerName(str, Enum):
     QBLOX_SPIRACK = "qblox_spi_rack"
     KEYSIGHT_E5080B = "keysight_e5080b_controller"
     AGILENT_E5071B = "agilent_e5071B_controller"
+    QMM = "qmm"
```

Comment:
Same here, I would prefer to give the full name descriptions. See other comments, too.

Reply:
done